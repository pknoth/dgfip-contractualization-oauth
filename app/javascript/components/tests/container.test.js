import ReactTestRenderer from 'react-test-renderer'
import Container from '../container'

describe('components | Container', () => {
  describe('render', () => {
    const props = {
      children: '<p>Some html</p>'
    }
    const renderer = ReactTestRenderer.create(<Container {...props} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
