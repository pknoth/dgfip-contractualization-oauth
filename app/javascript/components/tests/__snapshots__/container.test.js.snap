// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Container render should match snapshot 1`] = `
<div
  className="jsx-1157840685"
  style={
    Object {
      "maxWidth": 1200,
    }
  }
>
  &lt;p&gt;Some html&lt;/p&gt;
</div>
`;
